microsoft-edge/devtools-guide-chromium/beginners/html.md

```diff
@@ -262,7 +262,7 @@ You're almost done. You made a few changes to the page in DevTools, but they're
     
 1. Go back to the **editor tab** and replace the content of your `index.html` file with the code that you copied.
 
-   :::image type="content" source="media/beginners-html-copy2.msft.png" alt-text="How your index.html file should look." lightbox="media/beginners-html-copy2.msft.png":::
+   ![How your index.html file should look.](media/beginners-html-copy2.msft.png)
```

Comment:
```suggestion
   > [!div class="mx-imgBorder"]
   > ![How your index.html file should look.](media/beginners-html-copy2.msft.png)
```

Reply:
Let me suggest a different idea: this image already has 3 grey borders (top, left, right) because it's a capture of the full window, but the bottom window border is cut off. I'd prefer to add the bottom border in the image directly, instead of adding the div class=mx-imageBorder.
This div goes a bit against our goal of simplifying the markdown and only using standard image tagging.
I don't think we can do this for all images, but this one at least feels like the right thing to do. I'll take a look at others too.